src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Seamlessly adopt New Relic Service catalog with the GitHub integration
+tags:
+  - New Relic integrations
+  - GitHub integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Implementing a New Relic Service Catalog can unlock considerable benefits, but at present, it requires a considerable investment of time and resources. You have to take care of team orchestration, user provisioning, role assignments, and entity ownership delineation, which create a significant threshold for entry. To ease this challenge, we suggest the adoption of an automation solution that capitalizes on existing team configurations and relationships maintained in external systems such as GitHub. This simplifies the initial configuration, dramatically lessen the required effort, and accelerates the adoption of the New Relic Service Catalog, making its powerful insights and organization tools more accessible to users.
+
+<img
+  title="GitHub Integration dashboard"
+  alt="GitHub Integration dashboard"
+  src="/images/github-integration-screenshot.webp"
+/>
+
+<figcaption>
+After setting up our GitHub integration, see your data of teams, repos and users right out of the box.
+</figcaption>
+
+## Prerequisites [#prerequisites]
+
+Users need to ensure that they're logged in to their GitHub accounts. This needs to be the same GitHub account in which they want to install the GitHub app that New Relic will be providing during installation step.
+
+
+## Set up the GitHub integration [#set-up-github]
+
+Vist <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > + Integrations & Agents** > GitHub integration</DNT> tile and follow the below steps to complete the GitHub integration.
+
+<Steps>
+<Step>
+
+## Service Catalog Details [#service-catalog-details]
+
+In this installation step, we will be asked to enable/disable the Power Service catalog option. Power Service catalog provides the users the ability to import all the data related to Teams, Repositories, Pull Request, Pull Request Review and Deployment. 
```

Comment:
In this step, you have to enable or disable the Power Service catalog option. This option provides users the ability to import all data related to teams, repositories, pull requests, pull request reviews, and deployments.